src/mbgl/annotation/shape_annotation_impl.hpp

```diff
@@ -6,6 +6,7 @@
 #include <mbgl/util/geo.hpp>
 #include <mbgl/util/geojsonvt/geojsonvt.hpp>
 
+#include <memory>
```

Comment:
It's necessary for:

```
using Map = std::map<AnnotationID, std::unique_ptr<ShapeAnnotationImpl>>
```

This was compiling before due to transitive includes.